src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
+
+[assembly: TypesToLoad(typeof(MSTestDiscoverer), typeof(MSTestExecutor))]
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities
```

Comment:
Could we make the namespace simply `Microsoft.VisualStudio.TestPlatform` or `Microsoft.VisualStudio.TestPlatform.ObjectModel`? The `Utilities` part of the name seems verbose and I may not intuitively remember that the attribute needs to be created in that namespace...